game/templates/game/board.html

```
@@ -81,12 +81,10 @@
             <p style="color: #aaa; margin-bottom: 30px; font-size: 0.95rem;">Enter names and select a game mode.</p>
 
             <div id="nameInputs" style="display: flex; flex-direction: column; gap: 10px; margin-bottom: 25px;">
-                <input type="text" id="whiteNameInput" placeholder="White Player Name" maxlength="17" required
-                    minlength="1"
+                <input type="text" id="whiteNameInput" placeholder="White Player Name" maxlength="17" 
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)